packages/example/src/pages/contributing.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Contributing
```

Comment:
```suggestion
title: Contributions
```
We are using `contributions` on the Carbon site, we should keep them consistent.